common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala

```diff
@@ -0,0 +1,14 @@
+package com.gu.mediaservice.lib.collections
+
+import com.gu.mediaservice.model.Collection
+
+object CollectionsManager {
+  def onlyLatest(collections: List[Collection]): List[Collection] = {
```

Comment:
I was thinking this - but you do land up wondering if that should then exist on `collections`. e.g. 
```JSON
{
  "collections": {
    "data": [],
    "actionData": [
      { "action": "edit", "date": "...", "author": "me@you.com" },
      ...
    ]
  }
}
```